aries-site/src/pages/templates/filtering.mdx

```diff
@@ -0,0 +1,111 @@
+import { Example } from '../../layouts';
+import {
+  FilteringCards,
+  FilteringLists,
+  FilteringWithRangeSelector,
+  FilteringWithDropButton,
+  FilteringWithSelect,
+  PersistentFiltering,
+} from '../../examples';
+
+<Example
+  code="https://raw.githubusercontent.com/hpe-design/design-system/master/aries-site/src/examples/templates/filtering/PersistentFiltering.js"
+  figma="https://www.figma.com/file/EA0OZFdocKVcQ55c7Jainb/HPE-Filter-Templates?node-id=681%3A1430"
+  showResponsiveControls={['laptop', 'mobile']}
+  template
+>
+  <PersistentFiltering />
+</Example>
+
+## What makes up filtering?
+
+Filtering allows data to be narrowed down around a set of specified parameters. When users may not have a specific record in mind but are seeking to group data around a certain set of attributes, filtering should be used to allow the user to narrow the data and take action on relevant records.
+
+If a user will likely be looking for a specific record as opposed to groups of data, consider implementing [search](/templates/searching).
```

Comment:
What do we want to do with this since we do not have a "searching" page? Do we want to remove the link for now and create an issue for it?

Reply:
Yes, I think removing the link would be the right move for now.